enterprise/frontend/src/embedding-sdk/components/private/InteractiveQuestionResult/InteractiveQuestionResult.tsx

```diff
@@ -49,20 +52,35 @@ export const InteractiveQuestionResult = ({
     onSave,
     isSaveEnabled,
     saveToCollectionId,
+    isCardIdError,
   } = useInteractiveQuestionContext();
 
   const [isSaveModalOpen, { open: openSaveModal, close: closeSaveModal }] =
     useDisclosure(false);
 
   // When visualizing a question for the first time, there is no query result yet.
-  const isQueryResultLoading = question && !queryResults;
+  const isQueryResultLoading =
+    question && shouldRunCardQuery(question) && !queryResults;
```

Comment:
When we fail to resolve an entity ID, there won't be any question loaded, so we need to use the same logic to check that. But `question` will be populated, from my testing this is needed so `CreateQuestion` still works, it works by populating an empty question.

https://github.com/metabase/metabase/blob/c3b6176b8ecf05cfe3e938811863a8cf3f040fdb/enterprise/frontend/src/embedding-sdk/lib/interactive-question/run-question-query.ts#L32-L38

This code is basically the same, but the condition has been moved into this new function.

If we don't add this condition, the component will be stuck in the loading state.